x-pack/plugins/siem/server/lib/detection_engine/signals/filter_events_with_list.test.ts

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import uuid from 'uuid';
+import { filterEventsAgainstList } from './filter_events_with_list';
+import { mockLogger, repeatedSearchResultsWithSortId } from './__mocks__/es_results';
+
+import { ListClient } from '../../../../../lists/server';
+import { ListItemArraySchema } from '../../../../../lists/common/schemas';
+
+describe('filterEventsAgainstList', () => {
```

Comment:
Do you also want to add a test for when the `operator` is `included`?

Reply:
I'm wondering how that would look from a signal generation standpoint? If an event matches a rule, and it also matches a list that we want to generate a signal on, is that doing double the work for the same result? My understanding for lists was that if an event matches a rule but it's in an allowlist (some internal ip or something) then don't generate a signal. But if we just have lists that say make sure you generate a signal on this, isn't that just stating do the thing the rule was written to do? It's probable that I'm missing something from the extended talks on exception lists though.